tests/Fixtures/e2e/PhpUnit_Depends_Order/expected-output.txt

```diff
@@ -1,6 +1,6 @@
 Total: 1
-Killed: 0
-Errored: 1
+Killed: 1
+Errored: 0
```

Comment:
@maks-rafalko can you tell why there was an erroring mutation?

Reply:
I get this if I downgrade back again:
```
PHPUnit 6.5.14 by Sebastian Bergmann and contributors.

E
Fatal error: Uncaught Exception: One of the tests were skipped. Something wrong with the order of executed tests.
``` 
Is this all right that we stopped getting this error?

Reply:
As I see it this E2E test only supposed to test the initial run with a random order, nothing else.